pkg/loader/fileLoader.go

```diff
@@ -81,6 +82,14 @@ func (f *FileLoader) Start() (err error) {
 	if f.caPEM, err = os.ReadFile(f.ca); err != nil {
 		return errors.Wrapf(err, "cannot read ca file %s", f.ca)
 	}
+	_, err = os.Stat(f.cert)
```

Comment:
This is already done in isNew() function. i think we have to figure out the problem more deeply...